build.gradle

```diff
@@ -248,7 +259,15 @@ dependencies {
     testContractImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test'
     testImplementation  group: 'au.com.dius', name: 'pact-jvm-provider-spring_2.12', version: '3.6.14'
     testContractImplementation  group: 'au.com.dius', name: 'pact-jvm-provider-spring_2.12', version: '3.6.14'
+
+  // consumer pact
     testContractImplementation("org.junit.jupiter:junit-jupiter-api:5.5.2")
+  // consumer pact - TBD - Pact Provider version and
```

Comment:
do we need this comment , the dependency look self explanatory

Reply:
These comments have now been removed